packages/redis/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# @openfn/language-redis
+
+## 1.0.0
+
+Add the following operations
+
+- get(key)
```

Comment:
Reminder to double check these signatures just before we release

Reply:
Also, I'll come back and do little docs pass on this when finished, but I think I'd like to say something like "this is a minimal, low level wrapper around the nodejs redis API". Just to try and capture the spirit of this first release.